src/Sdk/DTWebApi/WebApi/Exceptions.cs

```diff
@@ -2476,24 +2476,32 @@ protected UnresolvableActionDownloadInfoException(SerializationInfo info, Stream
             : base(info, context)
         {
         }
-    }
 
-    [Serializable]
-    public sealed class FailedToResolveActionDownloadInfoException : DistributedTaskException
```

Comment:
we can't change this file, these files are minor from service, we need to change the server to throw a different exception.